cuebot/src/main/java/com/imageworks/spcue/dao/postgres/ProcDaoJdbc.java

```diff
@@ -264,30 +264,25 @@ public void updateProcMemoryUsage(FrameInterface f, long rss, long maxRss,
                     "SELECT pk_frame FROM proc WHERE pk_frame=? FOR UPDATE",
                     String.class, f.getFrameId()).equals(f.getFrameId())) {
 
-                getJdbcTemplate().update(UPDATE_PROC_MEMORY_USAGE,
-                        rss, maxRss, vss, maxVss,
-                        usedGpuMemory, maxUsedGpuMemory, f.getFrameId());
-            }
```

Comment:
Alright, this is was a weird merge issue. Not sure where it came from. Good catch.